flow/connectors/snowflake/snowflake.go

```diff
@@ -604,28 +606,19 @@ func (c *SnowflakeConnector) CreateRawTable(ctx context.Context, req *protos.Cre
 	}
 
 	if !schemaExists.Valid || !schemaExists.Bool {
-		_, err := c.database.ExecContext(ctx, fmt.Sprintf(createSchemaSQL, c.rawSchema))
+		_, err := c.execWithLogging(ctx, fmt.Sprintf(createSchemaSQL, c.rawSchema), nil)
 		if err != nil {
 			return nil, err
 		}
 	}
-
-	createRawTableTx, err := c.database.BeginTx(ctx, nil)
```

Comment:
why removing tx?

Reply:
Snowflake doesn't have transactional DDL, and this tx is for a single statement anyway